test/unit/sthDatabase_test.js

```diff
@@ -723,372 +825,535 @@ function retrievalTest(params, options, count, done) {
+                    dateFrom: new Date(DATE.getTime() + 1000 * 60 * 60)
+                },
+                0
+            )
         );
 
-        it('should retrieve ' + count + ' ' + aggregation + ' ' +
-          dataType + ' data for a resolution of ' + aggregationBy +
-          ' and an aggregation method of min if ' + count + ' data is inserted',
-          retrievalTest.bind(
-            null,
-            {
-              aggregation: aggregation,
-              dataType: dataType
-            },
-            {
-              aggrMethod: 'min',
-              aggrPeriod: aggregationBy
-            },
-            count
-          )
+        it(
+            'should retrieve ' +
+                count +
+                ' ' +
+                aggregation +
+                ' ' +
+                dataType +
+                ' data with dateTo if ' +
+                count +
+                ' data is inserted',
+            retrievalTest.bind(
+                null,
+                {
+                    aggregation: aggregation,
+                    dataType: dataType
+                },
+                {
+                    dateTo: new Date()
+                },
+                count
+            )
         );
 
-        it('should retrieve ' + count + ' ' + aggregation + ' ' +
-          dataType + ' data for a resolution of ' + aggregationBy +
-          ' and an aggregation method of max if ' + count + ' data is inserted',
-          retrievalTest.bind(
-            null,
-            {
-              aggregation: aggregation,
-              dataType: dataType
-            },
-            {
-              aggrMethod: 'max',
-              aggrPeriod: aggregationBy
-            },
-            count
-          )
+        it(
+            'should retrieve ' +
+                0 +
+                ' ' +
+                aggregation +
+                ' ' +
+                dataType +
+                ' data with dateTo if previous if ' +
+                count +
+                ' data is inserted',
+            retrievalTest.bind(
+                null,
+                {
+                    aggregation: aggregation,
+                    dataType: dataType
+                },
+                {
+                    dateTo: new Date(DATE.getTime() - 1000 * 60 * 60)
+                },
+                0
+            )
         );
-      } else {
-        it('should retrieve ' + count + ' ' + aggregation + ' ' +
-          dataType + ' data for a resolution of ' + aggregationBy +
-          ' and an aggregation method of occur if ' + count + ' data is inserted',
-          retrievalTest.bind(
-            null,
-            {
-              aggregation: aggregation,
-              dataType: dataType
-            },
-            {
-              aggrMethod: 'occur',
-              aggrPeriod: aggregationBy
-            },
-            count
-          )
+
+        it(
+            'should retrieve ' +
+                count +
```

Comment:
Excesive line break here and other places in this file. Please have a general look on it.

Reply:
Fixed  c00464eb